src/components/Header/HeaderBanner/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { AlertTriangle, Megaphone } from "lucide-react";
+import styles from "./header-banner.module.css";
+import { Twitch } from "@/assets/Icons";
+import React from "react";
+
+type Props = {
+  href?: string;
+  message: string;
+  onClick?: () => void;
+};
+
+const WarningBanner: React.FC<Props> = ({ href, message, onClick }) => {
+  return (
+    <div className={[styles.banner, styles.warning].join(" ")} onClick={onClick}>
+     <AlertTriangle className={styles.icon} /><a href={href} target="_blank">{message} Click here to know more.</a>
+    </div>
```

Comment:
We should avoid using `onClick` on `<div>`, instead a `<button>` would be a better option for this case. See https://www.w3schools.com/accessibility/accessibility_buttons_links.php and https://www.yanandcoffee.com/2020/08/10/button-vs-div/ if you wanna understand how to still use a `<div>` but with a button role.

Reply:
great point, i'm going to add `role="button"` to this div since it's not really a button. thanks!